src/scaffoldmaker/meshtypes/meshtype_3d_stomach1.py

```diff
@@ -1222,11 +1222,12 @@ def createStomachMesh3d(region, fm, coordinates, stomachTermsAlong, allAnnotatio
     elementsThroughEsophagusWall = GEJSettings['Number of elements through wall']
     ostiumDiameter = GEJSettings['Ostium diameter']
     limitingRidge = options['Limiting ridge']
+    wallThickness = options['Wall thickness'] * GEJSettings['Unit scale']
```

Comment:
It's unexpected that the wall thickness is multiplied by the unit scale of the GEJ. I suggest having the actual thickness in unit scale for the wall thickness, and dividing by GEJSettings['Unit scale'] when transferring to the GEJ in updateSubScaffoldOptions.
I hadn't thought about the GEJ options all needing rescalin , sorry. I think it's OK if they keep their custom unit scale (as you've done) as long as the parent wall thickness is independent as above.
(Also, in hindsight I would have aligned not just the fundus-body junction on the central path, but also it's direction at that point. Not asking you to do this as we've bigger fish to fry.)